src/Spectre.Angular2Client/src/app/plotly/plotly.module.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * plotly.module.ts
+ * Module with component wrapping Plotly
+ *
+   Copyright 2017 Grzegorz Mrukwa
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+
+import { PlotlyComponent } from './plotly.component';
+import { GuidService } from './guid.service';
+
+@NgModule({
+  providers: [
+    GuidService
+  ],
+  imports: [
+    CommonModule
+  ],
+  declarations: [
+      PlotlyComponent
+  ],
+  exports: [
+      PlotlyComponent
```

Comment:
@Sebaxp95 organized it in such way, that heatmap component exercises this one, while is not included in here.